### PR TITLE
[C-633] Polish a few docs pieces

### PR DESCRIPTION
**Pull request by raymondjacobson**
This is just the stuff that jumped out at me that I felt could be a bit nicer. Open to any feedback.

Some small structural things:
- Upgraded docusaurus to latest version on node 14 (gets us nice breadcrumbs)
- API => Developers
- Auto expand Developers left-nav
- Some copy fixes / images / code block structure

I got rid of the 0-index thing. It's kinda silly IMO!

Demo: ray.audius.co

### CODE REVIEW

File: typedoc-plugin-audius-theme/typedoc-plugin-audius-theme [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/ray/Documents/Audius/docs.audius.org/typedoc-plugin-audius-theme
```
Comment (sliptype) [inline typedoc-plugin-audius-theme/typedoc-plugin-audius-theme:1]: I think there might be something wrong with the `init-sdk-docgen` command creating this file for some reason

**Review by nicoback2 — APPROVED** [review]
Looks great!
Could you remove the Table of Contents in the body on this page? https://ray.audius.co/token/running-a-node/advanced_setup

File: docs/developers/sdk/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+- [HTML + JS](#html--js)
+
+### Node.js
+
+#### 1. Install the SDK package using your preferred JS package manager
+
+In your terminal, run:
+
+```bash"
+npm install web3 @audius/sdk
```
Comment (sliptype) [inline docs/developers/sdk/index.md:32]: Should we list web3 second?

```suggestion
npm install @audius/sdk web3
```

**Review by sliptype — APPROVED** [review]
Looks great! I noticed on the top level Developers page the JavaScript SDK tile just says "7 items", can we override that with some better text? 
image

File: docs/developers/sdk/index.md (hunk 2 above)
Comment (raymondjacobson) [reply docs/developers/sdk/index.md:32]: I'm good with either. I was thinking that since web3 is a "dep" maybe it should come first, but no strong opinion

Comment (sliptype) [reply docs/developers/sdk/index.md:32]: Yeah that makes sense, I'm fine with it as is!

File: typedoc-plugin-audius-theme/typedoc-plugin-audius-theme (hunk 1 above)
Comment (sliptype) [reply typedoc-plugin-audius-theme/typedoc-plugin-audius-theme:1]: Oh, seems like it's not idempotent. Running it multiple times causes it to create this file

Comment (raymondjacobson) [reply typedoc-plugin-audius-theme/typedoc-plugin-audius-theme:1]: fun! i can try to fix that on this PR

Comment (sliptype) [reply typedoc-plugin-audius-theme/typedoc-plugin-audius-theme:1]: Thanks! I don't know enough about symlinks to understand why this is happening haha

**Comment by raymondjacobson** [comment]
> Looks great! I noticed on the top level Developers page the JavaScript SDK tile just says "7 items", can we override that with some better text? image

Unfortunately I don't think there's a way to override. In the future we could roll our own index page here, but felt that this was at least kinda nice to have. Docusaurus does the same thing: https://docusaurus.io/docs/next/category/guides

**Pull request merged**